clmm/modbackend/numcosmo.py

```diff
@@ -0,0 +1,186 @@
+# NumCosmo implementation of CLMModeling
```

Comment:
Can you add docstrings to the fucntions here? If the API here work exactly as in `clmm/modbackend/func_layer.py`, you could just copy them.

Reply:
See above